app/services/rss_reader.rb

```diff
@@ -95,34 +87,16 @@ def make_from_rss_item(item, user, feed)
     article
   end
 
-  def get_host_without_www(url)
-    url = "http://#{url}" if URI.parse(url).scheme.nil?
-    host = URI.parse(url).host.downcase
-    host.start_with?("www.") ? host[4..] : host
-  end
-
-  def medium_reply?(item)
-    get_host_without_www(item.url.strip) == "medium.com" &&
-      !item[:categories] &&
-      content_is_not_the_title?(item)
-  end
-
-  def content_is_not_the_title?(item)
-    # [[:space:]] removes all whitespace, including unicode ones.
-    content = item.content.gsub(/[[:space:]]/, " ")
-    title = item.title.delete("…")
-    content.include?(title)
-  end
-
-  def article_exists?(user, item)
-    title = item.title.strip.gsub('"', '\"')
-    feed_source_url = item.url.strip.split("?source=")[0]
-    relation = user.articles
-    relation.where(title: title).or(relation.where(feed_source_url: feed_source_url)).exists?
-  end
-
   def report_error(error, metadata)
     Honeybadger.context(metadata)
     Honeybadger.notify(error)
   end
+
+  def get_item_count_error(feed)
```

Comment:
I know you just moved this, but maybe it would be a good opportunity to make this a bit more idiomatic Ruby:

1. Remove `get_` prefix.
2. Use guard clause.

```ruby
def item_count_error(feed)
  return "NIL FEED, INVALID URL" unless feed

  feed.entries ? feed.entries.length : "no count"
end
```

As a bonus, the strings could maybe become constants.

Reply:
Added 1 and 2, I haven't added the constants because I definitely want to change how we signal "invalid url" using symbols, not generic strings. Something like `:invalid_url` and `:zero_count`. Made a note of this as well

Reply:
Hm, the difference between symbols and frozen strings are minimal in this specific case (MRI implementation details aside): neither is resistant to typos while checking nor is there a significant difference in expressiveness. The name of the method makes me wonder if these should be actual errors, e.g. exception classes. "Number of entries or error message/symbol" is an unwieldy return type.

Reply:
ahhaha i agree, that method is a bit odd